conf/app.routes

```diff
@@ -25,11 +25,8 @@ GET         /applications/:id/resend-verification                   controllers.
 GET         /applications/:id/access/scopes                         controllers.ApplicationController.manageScopes(id: String)
 POST        /applications/:id/access/scopes                         controllers.ApplicationController.updateScopes(id: String)
 
-GET         /applications/:id/ip-whitelist                          controllers.ApplicationController.manageIpWhitelist(id: String)
-GET         /applications/:id/ip-whitelist/:whitelistedIp/remove    controllers.ApplicationController.removeWhitelistedIp(id: String, whitelistedIp: String)
-POST        /applications/:id/ip-whitelist/:whitelistedIp/remove    controllers.ApplicationController.removeWhitelistedIpAction(id: String, whitelistedIp: String)
-GET         /applications/:id/ip-whitelist/add                      controllers.ApplicationController.addWhitelistedIp(id: String)
-POST        /applications/:id/ip-whitelist/add                      controllers.ApplicationController.addWhitelistedIpAction(id: String)
+GET         /applications/:id/ip-whitelist/update                   controllers.ApplicationController.updateWhitelistedIp(id: String)
```

Comment:
Why a GET to `/update`?

Reply:
Might be tempted to rename the function in `ApplicationController` to make it a little clearer that we're fetching a page

Reply:
Yeah we could call it updateWhitelistedIpPage.